### PR TITLE
added runTimeout fix, also, pending = implement loading... text to di…

### PR DESCRIPTION
**Pull request by KalaRaja**
added runTimeout fix,
pending = implement loading... text to display on endline screen until project loading is completed.

### CODE REVIEW

File: packages/client-mav/src/app/endline/display-screen/display-screen.component.ts [hunk 1]
```diff
@@ -24,15 +26,36 @@ export class DisplayScreenComponent implements OnInit {
...
+       * show 'Waiting for Runner' text.
+       * To do this, a timeout is fetched from the environment and when the timeout expires,
+       * a SignupStartedMessage is simulated, this ends the timer and displays
+       * 'Waiting for Runner' text.
+       */
+      if (msg instanceof RunStartedMessage) {
+        timeoutHandler = setTimeout(() => {
+          this.timerService.stop();
+          this.handleMessage(this.createDummySignupStartedMessage());
+        }, environment.projectConfiguration.runTimout);
```
Comment (bherr2) [inline packages/client-mav/src/app/endline/display-screen/display-screen.component.ts]: Environment should not be used. Use this.dataService.config.runTimeout.

File: packages/client-mav/src/app/endline/display-screen/display-screen.component.ts [hunk 2]
```diff
@@ -24,15 +26,36 @@ export class DisplayScreenComponent implements OnInit {
   constructor(private dataService: XMacroscopeDataService, private timerService: TimerService) {}
 
   ngOnInit() {
+    let timeoutHandler: any; // used to store setTimeout Id to clear it when a new message arrives.
```
Comment (bherr2) [inline packages/client-mav/src/app/endline/display-screen/display-screen.component.ts]: timeoutHandler seems like a weird name. Maybe you meant timeoutHandle?

File: packages/client-mav/src/app/endline/display-screen/display-screen.component.ts [hunk 3]
```diff
@@ -24,15 +26,36 @@ export class DisplayScreenComponent implements OnInit {
...
+      }
      });
+
     this.timerService.getFormattedTimeObservable().subscribe((timerText) => {
       this.timerText = timerText;
     });
   }
 
   handleMessage(msg: Message) {
+    console.log(msg);
```
Comment (bherr2) [inline packages/client-mav/src/app/endline/display-screen/display-screen.component.ts]: Remove debug info

File: packages/client-mav/src/app/endline/display-screen/display-screen.component.ts [hunk 4]
```diff
@@ -54,4 +77,9 @@ export class DisplayScreenComponent implements OnInit {
       this.personaShape =  personAttributes.icon;
     }
   }
+
+  createDummySignupStartedMessage(): SignupStartedMessage {
+    return new SignupStartedMessage();
+
```
Comment (bherr2) [inline packages/client-mav/src/app/endline/display-screen/display-screen.component.ts]: Remove unnecessary blank lines (unless used for clarity in a block of code)

File: packages/client-mav/src/environments/environment.deploy.ts [hunk 5]
```diff
@@ -2,6 +2,7 @@ export const environment = {
   production: true,
   useHashStrategy: true,
   projectConfiguration: {
+    runTimout: 10000,
```
Comment (bherr2) [inline packages/client-mav/src/environments/environment.deploy.ts]: move all to just below defaultUsState

**Comment by KalaRaja** [comment]
applied all review comments and suggestions to the code.

**Pull request merged**